Changelog.md

```diff
@@ -1,6 +1,8 @@
 ### 0.9.0 (unreleased)
 
 Breaking changes:
+ * Disallow ``.pop()`` on arrays containing nested mappings.
+ * Disallow ``delete`` on types that contain nested mappings.
```

Comment:
Is this in alphabetic order :-D? I'm honestly not sure :-D. [no need to do anything as far as I'm concerned]

Reply:
It is. arrays vs types ,also originally done by vim using `:sort` (that's why we had that rebase artefact, it reordered another change, too)